runelite-client/src/main/java/net/runelite/client/plugins/party/messages/StatusUpdate.java

```diff
@@ -0,0 +1,27 @@
+package net.runelite.client.plugins.party.messages;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import net.runelite.client.party.messages.PartyMemberMessage;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+public class StatusUpdate extends PartyMemberMessage
+{
+	
+	// we use boxed fields to null out properties that haven't changed
+	// and minified field names to reduce the payload size
```

Comment:
you can use `@SerializedName` to achieve this without having to mangle the source as much